### DISCUSSION TITLE
Breadcrumbs generation doesn't care parent page across multiple collections

### DISCUSSION THREAD

**Discussion by mumincelal** [payloadcms/plugin-nested-docs#22] [Ideas]
I am using the nested-docs plugin to create breadcrumbs. 4 different collections use nested-docs but when I select the parent page from one collection to another, the plugin doesn't create breadcrumbs correctly. It doesn't care parent page but if I select the parent page in the same collection then the plugin is successful to create breadcrumbs correctly.

Here is the setup:

```
nestedDocs({
   collections: [Page.slug, Post.slug, EnBlog.slug, TrBlog.slug],
   parentFieldSlug: 'parent',
   breadcrumbsFieldSlug: 'breadcrumbs',
   generateLabel: (_, page) => (page.name as string) || '',
   generateURL: (pages) =>
     pages.reduce((url, page) => ${url}/${page.slug}, '')
})
```

If I choose a parent from page collection to page collection, the plugin works correctly but If I choose a parent from post collection to page collection then the plugin can not generate parent page breadcrumbs.

The plugin needs to be worked across multiple collections.

**Comment by jacobsfletch** [comment]
Related conversation: https://discord.com/channels/967097582721572934/1115245109185880074

**Comment by jacobsfletch** [comment]
Hey @mumincelal since this is more of a feature request, I'm converted this issue to a discussion so we can keep the conversation going here!

**Reply by mumincelal** [reply]
Sure. Thank you

**Comment by mumincelal** [comment]
@jacobsfletch Is there any plan to implement it?

**Reply by ClementLaval** [reply]
@mumincelal  I need the same thing 👍
I just made a proposal PR.
